base/server/python/pki/server/cli/__init__.py

```diff
@@ -90,6 +90,11 @@
     r'[^-]*-group-member-find$',
     r'[^-]*-group-member-add$',
     r'[^-]*-group-member-del$',
+    r'[^-]*-acl-find$',
+    r'[^-]*-acl-add$',
+    r'[^-]*-acl-del$',
+    r'[^-]*-crl-record-show$',
+    r'[^-]*-crl-record-cert-find$',
```

Comment:
The `<subsystem>-crl-record-*` is implemented only for the [CA](https://github.com/dogtagpki/pki/tree/master/base/ca/src/main/java/org/dogtagpki/server/ca/cli) and not for the other subsystems. The python code was raising an error if the instance does not contain the CA and the command was implemented only for the CA.
Should this be limited to the CA? Are CRLs handled by other systems?

Reply:
Ah yes, let me fix that.

Reply:
It should be fixed now.